src/basic/os-util.c

```diff
@@ -19,6 +19,27 @@
 #include "utf8.h"
 #include "xattr-util.h"
 
+/* Helper struct for naming simplicity and reusability */
+static const struct {
+        const char *strict_xattr_name;
+        const char *release_file_directory;
+        const char *release_file_path_prefix;
+        const char *release_file_name_prefix;
+} image_class_release_info[_IMAGE_CLASS_MAX] = {
+        [IMAGE_EXTENSION] = {
+                .strict_xattr_name = "user.extension-release.strict",
+                .release_file_directory = "/usr/lib/extension-release.d/",
+                .release_file_path_prefix = "/usr/lib/extension-release.d/extension-release.",
+                .release_file_name_prefix = "extension-release.",
+        },
+        [IMAGE_CONFEXT] = {
+                .strict_xattr_name = "user.confext-release.strict",
+                .release_file_directory = "/etc/confext-release.d/",
+                .release_file_path_prefix = "/etc/confext-release.d/confext-release.",
+                .release_file_name_prefix = "confext-release.",
```

Comment:
so we actually prefer to use the same prefixes, xattr and name across the different extensions, with the only difference being the path prefix - /etc/ vs /usr/lib/ that way the extension file is common to all extension images types